optimization/benchmarks/knapsack.py

```diff
@@ -0,0 +1,85 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""knapsack benchmarks"""
+
+import random
+
+from qiskit import Aer
+from qiskit.algorithms import QAOA, VQE
+from qiskit.algorithms.optimizers import COBYLA
+from qiskit.circuit.library import EfficientSU2
+from qiskit.utils import algorithm_globals, QuantumInstance
+
+from qiskit_optimization.algorithms import MinimumEigenOptimizer, GroverOptimizer
+from qiskit_optimization.applications import Knapsack
+from qiskit_optimization.converters import QuadraticProgramToQubo
+
+# pylint: disable=redefined-outer-name, invalid-name, attribute-defined-outside-init
+
+
+class KnapsackBenchmarks:
+    """Knapsack benchmarks"""
+
+    num_items = [2, 4, 8, 12]
+
+    params = ([2, 4, 8, 16], [8, 16, 32, 64])
```

Comment:
Could you adjust param numbers so that each benchmark takes at most a couple of seconds?